Userland/Utilities/image.cpp

```diff
@@ -43,8 +44,10 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         bytes = TRY(Gfx::PNGWriter::encode(*frame));
     } else if (out_path.ends_with(".qoi"sv, CaseSensitivity::CaseInsensitive)) {
         bytes = TRY(Gfx::QOIWriter::encode(*frame));
+    } else if (out_path.ends_with(".ppm"sv, CaseSensitivity::CaseInsensitive)) {
+        bytes = TRY(Gfx::PortableFormatWriter::encode(*frame));
```

Comment:
This was once sorted alphabetically, please keep it that way :)

Reply:
:hyperlinus: